src/components/SongCard/SongCard.js

```diff
@@ -1,28 +1,59 @@
-import React from 'react';
-import '../SongCard/SongCard.scss'
+import { React, Component } from 'react';
+import '../SongCard/SongCard.scss';
+import Lyric from '../Lyric/Lyric';
+import { GiMicrophone } from "react-icons/gi"
+import { fetchLyrics } from "../../APICalls";
+import { formatLyrics } from '../../utility';
 
+class SongCard extends Component {
+  constructor(props) {
+  super(props)
+  this.state = {
+    lyrics: {},
+    error: ''
+  }
+}
 
-const SongCard = ({ id, title, artist, genres, album_cover, handleSong, buttonIcon, isActive }) => {
+getLyrics = (artist, songtitle) => {
+  fetchLyrics(artist, songtitle)
+  .then(data => {
+    const formattedLyrics = formatLyrics(data, songtitle)
+    this.setState({lyrics: formattedLyrics})
+  })
+  .catch(error => this.setState({error: "We weren't able to get you those lyrics at this time"}))
```

Comment:
I think a good message would be `Lyrics unavailable`

Reply:
OK cook, I will make that change